### ISSUE TITLE
exo ssh -> panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by SebastienPittet** [exoscale/cli#513]
Here is what I got when I try to ssh a machine.

`exo compute instance ssh debian01 -z ch-gva-2`

My version is `exo 1.69.0 3ccaabf (egoscale 0.100.1)`

The result is:

**panic: runtime error: invalid memory address or nil pointer dereference**
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1085e3d]

goroutine 1 [running]:
github.com/exoscale/cli/cmd.(*instanceSSHCmd).cmdRun(0xc00016e100, 0xc0009bdce8?, {0xc0009bdcc0?, 0x0?, 0x0?})
        github.com/exoscale/cli/cmd/instance_ssh.go:105 +0x31d
github.com/spf13/cobra.(*Command).execute(0xc00047e280, {0xc00070ec30, 0x3, 0x3})
        github.com/spf13/cobra@v1.3.0/command.go:856 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0x203a180)
        github.com/spf13/cobra@v1.3.0/command.go:974 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.3.0/command.go:902
github.com/exoscale/cli/cmd.Execute({0x18a5c20?, 0x0?}, {0x18a6268?, 0xc0000061a0?})
        github.com/exoscale/cli/cmd/root.go:87 +0x218
main.main()
        github.com/exoscale/cli/main.go:34 +0x46

**Comment by jessicatoscani** [comment]
Hello @SebastienPittet,
Thanks for reporting this issue. 

Could you share more details please ? OS used, debug logs (`EXOSCALE_TRACE=1  exo compute instance ssh debian01 -z ch-gva-2`) and instance ID. 
You can send them to support@exoscale.com (or to me directly :)

**Comment by zyegfryed** [comment]
FWIW I'm unable to reproduce:

```shell
❯ exo version
exo 1.69.0 3ccaabf (egoscale 0.100.1)
❯ exo compute instance create gh-513
 ✔ Creating instance "gh-513"... 19s
┼──────────────────────┼──────────────────────────────────────┼
│   COMPUTE INSTANCE   │                                      │
┼──────────────────────┼──────────────────────────────────────┼
│ ID                   │ eafb680d-9ed1-4b38-bd6c-696f3f6c10d4 │
│ Name                 │ gh-513                               │
│ Creation Date        │ 2023-05-31 12:07:26 +0000 UTC        │
│ Instance Type        │ standard.medium                      │
│ Template             │ Linux Ubuntu 22.04 LTS 64-bit        │
│ Zone                 │ ch-gva-2                             │
│ Anti-Affinity Groups │ n/a                                  │
│ Security Groups      │ default                              │
│ Private Instance     │ No                                   │
│ Private Networks     │ n/a                                  │
│ Elastic IPs          │ n/a                                  │
│ IP Address           │ 85.217.160.186                       │
│ IPv6 Address         │ -                                    │
│ SSH Key              │ -                                    │
│ Disk Size            │ 50 GiB                               │
│ State                │ running                              │
│ Labels               │ n/a                                  │
│ Reverse DNS          │                                      │
┼──────────────────────┼──────────────────────────────────────┼
❯ exo compute instance ssh gh-513 -z ch-gva-2
The authenticity of host '85.217.160.186 (85.217.160.186)' can't be established.
ED25519 key fingerprint is SHA256:cJy56i4dI8xASr9yLCZVfeH3SFwtRiCsu8PCq0EBwJs.
This key is not known by any other names.
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Warning: Permanently added '85.217.160.186' (ED25519) to the list of known hosts.
Welcome to Ubuntu 22.04.2 LTS (GNU/Linux 5.15.0-72-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

  System information as of Wed May 31 12:08:06 UTC 2023

  System load:  0.9365234375      Processes:             110
  Usage of /:   3.8% of 48.27GB   Users logged in:       0
  Memory usage: 6%                IPv4 address for eth0: 85.217.160.186
  Swap usage:   0%

Expanded Security Maintenance for Applications is not enabled.

0 updates can be applied immediately.

Enable ESM Apps to receive additional future security updates.
See https://ubuntu.com/esm or run: sudo pro status


The list of available updates is more than a week old.
To check for new updates run: sudo apt update

ubuntu@gh-513:~$
logout
Connection to 85.217.160.186 closed.
```

OS: macOS Ventura 13.4 (22F66)
Arch: Apple M2 Pro (arm64)

**Comment by jessicatoscani** [comment]
I managed to reproduce it with a private instance. It will be fixed soon. @SebastienPittet Is it the same pb  ?

**Comment by jessicatoscani** [comment]
The panic will be fixed by #514
